src/templates/person.js

```diff
@@ -83,7 +85,7 @@ const Person = ({
       >
         <div className="flex flex-wrap mb-5 md:mx-2 person-content">
           <div className="sm:w-full lg:w-1/4 xl:w-1/6">
-            {!!profileImage && (
+            {!!profileImage && !!sketchImage && (
```

Comment:
any chance a person only have a professional picture and no sketch?

Reply:
Nope, because it is the requirement in the SugarLearning task to create your profile.
- you ask to take your picture (professional and silly).
- send the photo to the designers, and they will create both images for you.